griptape/drivers/web_scraper/markdownify_web_scraper_driver.py

```diff
@@ -23,9 +27,7 @@ class MarkdownifyWebScraperDriver(BaseWebScraperDriver):
     """
 
     include_links: bool = field(default=True, kw_only=True)
-    exclude_tags: list[str] = field(
-        default=Factory(lambda: ["script", "style", "head", "header", "footer", "svg"]), kw_only=True
-    )
+    exclude_tags: list[str] = field(default=Factory(lambda: DEFAULT_EXCLUDE_TAGS), kw_only=True)
```

Comment:
Will need to add `takes_self=True` after moving `DEFAULT_EXCLUDE_TAGS` into class.